include/deal.II/base/vectorization.h

```diff
@@ -526,6 +527,13 @@ class VectorizedArray
     data = base_ptr[offsets[0]];
   }
 
+  DEAL_II_ALWAYS_INLINE
+  void
+  gather(const std::array<Number *, width> &base_ptr, const unsigned int i)
+  {
+    data = base_ptr[i][0];
```

Comment:
why do you not need a loop over ``width`` here?

Reply:
This is the specialization for `static const unsigned int n_array_elements = 1;`.